restapi/src/main/java/io/stargate/web/docsapi/exception/ErrorCode.java

```diff
@@ -63,6 +60,13 @@
           "The parameter `page-size` is limited to %d.",
           DocsApiConfiguration.DEFAULT.getMaxPageSize())),
 
+  DOCS_API_GENERAL_UPGRADE_INVALID(
+      Response.Status.BAD_REQUEST, "The collection cannot be upgraded in given manner."),
```

Comment:
nit: why `GENERAL` in the enum name?

Reply:
No idea, to separate from `DOCS_API_SEARCH` for example..

Reply:
ok